test/unit/math/mix/scal/fun/Phi_approx_test.cpp

```diff
@@ -1,4 +1,5 @@
 #include <stan/math/mix/scal.hpp>
+#include <stan/math/mix/mat.hpp>
```

Comment:
Thanks for going through all the includes!

- Bob

> On Apr 27, 2016, at 6:30 PM, Daniel Lee <notifications@github.com> wrote:
> 
> In test/unit/math/mix/scal/fun/Phi_approx_test.cpp:
> 
> > @@ -1,4 +1,5 @@
> >  #include <stan/math/mix/scal.hpp>
> > +#include <stan/math/mix/mat.hpp>
> 
> this shouldn't include a mix header.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>